libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/utils/NavigationRouteEx.kt

```diff
@@ -0,0 +1,10 @@
+@file:JvmName("NavigationRouteEx")
+
+package com.mapbox.navigation.base.internal.utils
```

Comment:
Could you move this to `com.mapbox.navigation.base.internal.route.NavigatioRouteEx` to avoid the duplication?